sycl/plugins/unified_runtime/ur/adapters/cuda/queue.cpp

```diff
@@ -140,6 +142,11 @@ urQueueCreate(ur_context_handle_t hContext, ur_device_handle_t hDevice,
       if (URFlags & UR_QUEUE_FLAG_OUT_OF_ORDER_EXEC_MODE_ENABLE) {
         IsOutOfOrder = true;
       }
+      if (URFlags & UR_QUEUE_FLAG_PRIORITY_HIGH) {
+        UR_CHECK_ERROR(cuCtxGetStreamPriorityRange(nullptr, &Priority));
+      } else if (URFlags & UR_QUEUE_FLAG_PRIORITY_LOW) {
+        UR_CHECK_ERROR(cuCtxGetStreamPriorityRange(&Priority, nullptr));
```

Comment:
These might need a `ScopedContext` somewhere to ensure the correct CUDA context is active

Reply:
Thank. Added.